src/modules/auth/ui/views/sign-up-view.tsx

```
@@ -0,0 +1,224 @@
+ 
+   }
+  return (
+   <div className="flex flex-col gap-6">
+  <Card className="overflow-hidden p-0">
+    <CardContent className="grid p-0 md:grid-cols-2">
+        <Form {...form}>
+      <form onSubmit={form.handleSubmit(onSubmit)}className="p-6 md:p-8">
+        <div className="flex flex-col gap-6">
+            <div className="flex flex-col items-center text-center">
+                <h1 className="text-2xl font-bold">
+                     Let's Get Started
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)